$NetBSD: patch-aa,v 1.6 2022/12/07 08:15:59 jperkin Exp $

add extra options for cases where PREFIX is not in a standard location

add INSTALLEXADIR for some pkgsrc magic

$NetBSD: patch-aa,v 1.6 2022/12/07 08:15:59 jperkin Exp $

--- configure.server.orig	2019-04-22 18:20:02.000000000 +0000
+++ configure.server
@@ -17,6 +17,8 @@ do
   To configure Xymon, the following options are available:
      --rrdinclude DIRECTORY   : Specify location of RRDtool include files
      --rrdlib DIRECTORY       : Specify location of RRDtool libraries
+     --pngconfig PROGRAM      : Specify location of libpng-config program
+     --zlib DIRECTORY         : Specify location of Z library
      --pcreinclude DIRECTORY  : Specify location of PCRE include files
      --pcrelib DIRECTORY      : Specify location of PCRE libraries
      --sslinclude DIRECTORY   : Specify location of OpenSSL include files
@@ -25,6 +27,7 @@ do
      --ldaplib DIRECTORY      : Specify location of OpenLDAP libraries
      --caresinclude DIRECTORY : Specify location of C-ARES include files
      --careslib DIRECTORY     : Specify location of C-ARES libraries
+     --snmpconfig PROGRAM     : Specify location of net-snmp-config program
      --fping FILENAME         : Specify location of the Fping program
 
   The script will search a number of standard directories for
@@ -38,6 +41,12 @@ EOF
 	  "--rrdlib")
 	  	USERRRDLIB="$1"; shift
 		;;
+	  "--pngconfig")
+	  	USERPNGCONFIG="$1"; shift
+		;;
+	  "--zlib")
+	  	USERZLIB="$1"; shift
+		;;
 	  "--pcreinclude")
 	  	USERPCREINC="$1"; shift
 		;;
@@ -62,6 +71,9 @@ EOF
 	  "--careslib")
 	  	USERCARESLIB="$1"; shift
 		;;
+	  "--snmpconfig")
+	  	USERSNMPCONFIG="$1"; shift
+		;;
 	  "--fping")
 	  	USERFPING="$1"; shift
 		;;
@@ -404,14 +416,16 @@ then
 fi
 echo ""; echo ""
 
-echo "# Toplevel Makefile for Xymon"    > Makefile
+echo "# Toplevel Makefile for Xymon"     > Makefile
 
 echo "BUILDTOPDIR=\`pwd\`"               >>Makefile
 echo ""                                  >>Makefile
-echo "# configure settings for Xymon"   >>Makefile
+echo "# configure settings for Xymon"    >>Makefile
 echo "#"                                 >>Makefile
 echo "# Toplevel dir"                    >>Makefile
 echo "XYMONTOPDIR = $XYMONTOPDIR"        >>Makefile
+echo "# Linkfarm"                        >>Makefile
+echo "XYMONHOME = $XYMONHOME"            >>Makefile
 echo "# Server data dir for hist/ etc."  >>Makefile
 echo "XYMONVAR = $XYMONVAR"              >>Makefile
 echo "# CGI scripts go in CGIDIR"        >>Makefile
@@ -453,7 +467,7 @@ then
 	echo "CARESINCDIR = -I$CARESINC" >>Makefile
     fi
     if test "$CARESLIB" != ""; then
-	echo "CARESLIBS = -L$LIB -lcares" >>Makefile
+	echo "CARESLIBS = -L$CARESLIB -lcares" >>Makefile
 	echo "RPATHVAL += ${LIB}"        >>Makefile
     else
 	echo "CARESLIBS = -lcares"       >>Makefile
@@ -531,6 +545,7 @@ echo "LIBRTDEF = $LIBRTDEF"
 echo ""                                  >>Makefile
 echo "# Net-SNMP settings"               >>Makefile
 echo "DOSNMP = $DOSNMP"                  >>Makefile
+echo "SNMPCONFIG = $SNMPCONFIG"          >>Makefile
 echo ""                                  >>Makefile
 
 echo "# Large File Support settings"     >>Makefile
@@ -561,6 +576,9 @@ fi
 if test "$INSTALLETCDIR" != ""; then
 	echo "INSTALLETCDIR = $INSTALLETCDIR"   >>Makefile
 fi
+if test "$INSTALLEXADIR" != ""; then
+	echo "INSTALLEXADIR = $INSTALLEXADIR"   >>Makefile
+fi
 if test "$INSTALLEXTDIR" != ""; then
 	echo "INSTALLEXTDIR = $INSTALLEXTDIR"   >>Makefile
 fi
